average.js

```diff
@@ -1,6 +1,6 @@
 
 function average(numbers) {
-  return numbers.reduce((p, c)=> p + c, 0) / numbers.length;
+  numbers = numbers.filter(n => !Number.isNaN(n));
```

Comment:
Nice translation to concise code :)
The variable `numbers` is used in multiple contexts - at one place to mean all the numbers, at some places to mean the filtered numbers.
Try giving a new name for the filtered numbers, so that readers do not have confusion on the context while reading the code.